### PR TITLE
Introduced documentation help button organism.

### PR DESCRIPTION
**Pull request by novykh**
Based on this: [alternative 3 design](https://www.figma.com/file/BETHLn9ww9Jv5XyS0yaXwi/Need-help---community-%26-documentation-%26-faq-links-%231527?node-id=478%3A21).
Fixes [#1527](https://github.com/netdata/product/issues/1527)

Screenshot 2021-01-05 at 7 10 29 PM

Screenshot 2021-01-05 at 7 10 36 PM

Screenshot 2021-01-05 at 7 10 42 PM

### CODE REVIEW

File: src/components/icon/assets/unknownError.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="19" height="18" viewBox="0 0 19 18" xmlns="http://www.w3.org/2000/svg">
+<path d="M13.7599 3H5.2399C3.1899 3 1.9499 5.27 3.0599 6.99L7.5399 13.95C7.9999 14.66 8.7499 15.02 9.4999 15.02C10.2499 15.02 10.9999 14.66 11.4599 13.95L15.9399 6.99C17.0399 5.27 15.8099 3 13.7599 3ZM14.6699 6.18L10.1999 13.14C9.9799 13.48 9.6399 13.52 9.4999 13.52C9.3599 13.52 9.0199 13.48 8.7999 13.14L4.3299 6.18C4.0099 5.69 4.1999 5.24 4.2899 5.07C4.3799 4.9 4.6599 4.5 5.2499 4.5H13.7599C14.3399 4.5 14.6199 4.9 14.7199 5.07C14.8099 5.24 14.9899 5.69 14.6699 6.18ZM9.4999 9.25C9.0899 9.25 8.7499 9.59 8.7499 10C8.7499 10.41 9.0899 10.75 9.4999 10.75C9.9099 10.75 10.2499 10.41 10.2499 10C10.2499 9.59 9.9099 9.25 9.4999 9.25ZM9.3199 5.94C8.8899 6.04 8.6099 6.47 8.7099 6.9L9.0699 8.5C9.1099 8.66 9.2299 8.79 9.3999 8.83C9.6299 8.88 9.8699 8.74 9.9199 8.5L10.2799 6.9C10.2999 6.79 10.3099 6.67 10.2799 6.55C10.1899 6.11 9.7599 5.84 9.3199 5.94Z" />
```
Comment (apostolidhs) [inline src/components/icon/assets/unknownError.svg:2]: Can we do something to align the sizes to 24px?

**Review by apostolidhs — APPROVED** [review]

**Review by allelos — APPROVED** [review]
📜 🆘

File: src/organisms/documentation/index.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+const Help = ({
+  documentationUrl = "https://learn.netdata.cloud/",
+  issuesUrl = "https://github.com/netdata/netdata/issues/new",
+  communityUrl = "https://community.netdata.cloud/c/support/13",
+  communitySupportUrl = "https://community.netdata.cloud/",
+}) => {
+  const [isOpen, toggle] = useToggle()
+
+  return (
+    <Flex background="mainBackgroundDisabled" width="100vw" height="100vh">
```
Comment (apostolidhs) [inline src/organisms/documentation/index.js:70]: I don't know exactly the business, do we need to occur the entire screen? It seems that a `<Fragment` is needed here

**Review by apostolidhs — APPROVED** [review]

File: src/organisms/documentation/index.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+              />
+            </Section>
+          </Container>
+        </Layer>
+      )}
+    </Flex>
+  )
+}
+
+export default Help
```
Comment (apostolidhs) [inline src/organisms/documentation/index.js:142]: The default export name should reflect to the folder name

**Review by jacekkolasa — APPROVED** [review]

**Pull request merged**